### PR TITLE
Add issue template for release QA

### PR DESCRIPTION
**Pull request by bassosimone**
Xref: https://github.com/ooni/probe/issues/862

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-qa-check-list.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    - [ ] make sure that the test performed work in background
+- [ ] run just one of the tests and verify the others didn't
+- [ ] results analysis of all the above
+    - [ ] check that the total count on top matches
+    - [ ] check that the total counts on top match
+    - [ ] check that the data usage shows a sane number, ex 36KB up, 18KB down
+    - [ ] check that the total runtime shows a sane number, ex. 2.16s
+    - [ ] check specific measurements
+        - [ ] check that everything is displayed properly in this screen
+        - [x] check that the runtime is sane ex 2.71s
```
Comment (hellais) [inline .github/ISSUE_TEMPLATE/release-qa-check-list.md]: This should be unchecked.

File: .github/ISSUE_TEMPLATE/release-qa-check-list.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    - [ ] check that the time is decreasing
+- [ ] run the test in foreground and then put the app in background
+    - [ ] make sure that the time is making progress
+    - [ ] make sure that you receive a notification
+    - [ ] make sure that the test performed work in background
+- [ ] run the test with just https://ooni.torproject.org/ and https://expired.badssl.com/
+- [ ] run the test with only some categories
+    - [ ] make sure you did only actually test these categories
+- [ ] run the test with a very short test runtime
+- [ ] run the test without uploading measurements and then perform a background upload
```
Comment (hellais) [inline .github/ISSUE_TEMPLATE/release-qa-check-list.md]: 👍

Comment (hellais) [reply .github/ISSUE_TEMPLATE/release-qa-check-list.md]: Maybe we can also add something about the counter of the upload decreasing?

File: .github/ISSUE_TEMPLATE/release-qa-check-list.md (hunk 1 above)
Comment (bassosimone) [reply .github/ISSUE_TEMPLATE/release-qa-check-list.md]: done!

File: .github/ISSUE_TEMPLATE/release-qa-check-list.md (hunk 2 above)
Comment (bassosimone) [reply .github/ISSUE_TEMPLATE/release-qa-check-list.md]: done!

**Comment by bassosimone** [comment]
Merged with the requested changes

**Pull request merged**